src/dashboard.tsx

```diff
@@ -296,6 +298,14 @@ export class URLInput extends Widget {
     return this._urlChanged;
   }
 
+  /**
+   * The in browser dashboard check for authenticated dashboards.
+   */
+  set browserDashboardCheck(value: boolean) {
```

Comment:
I think you'll need to also add a `get` function here to allow `this.browserDashboardCheck` to work.

Perhaps we should enable [this](https://eslint.org/docs/rules/accessor-pairs) linter rule.